try {
    nonExistantFunction();
} catch (error) {
    console.error(error);
}
